activation/activation.go

```diff
@@ -117,11 +118,12 @@ func (b *Builder) PublishActivationTx(epoch types.EpochId) error {
 		PositioningAtx: *posAtxId,
 	}
 
-	bytes, err := xdr.Marshal(challenge)
+	var w bytes2.Buffer
+	_, err = xdr.Marshal(&w, &challenge)
```

Comment:
refactor to ChallengeAsBytes and put it somewhere visible 
so others will use the same method for encoding